DcWebxdcWidget/DcWebxdcWidget.swift

```diff
@@ -0,0 +1,177 @@
+import WidgetKit
+import SwiftUI
+import DcCore
+
+struct Provider: TimelineProvider {
+    func placeholder(in context: Context) -> UsedWebxdcEntry {
+        let limit: Int
+        switch context.family {
+        case .systemSmall:
+            limit = 4
+        case .systemMedium:
+            limit = 7
+        default:
+            limit = 7
+        }
+
+        let apps = [
+            WebxdcApp(accountId: 0, chatId: 0, messageId: 0, image: UIImage(named: "checklist"), title: "checklist"),
+            WebxdcApp(accountId: 0, chatId: 0, messageId: 1, image: UIImage(named: "hello"), title: "hello"),
+            WebxdcApp(accountId: 0, chatId: 0, messageId: 6, image: UIImage(named: "packabunchas"), title: "packabunchas"),
+            WebxdcApp(accountId: 0, chatId: 0, messageId: 3, image: UIImage(named: "webxdc"), title: "webxdc"),
+            WebxdcApp(accountId: 0, chatId: 0, messageId: 2, image: UIImage(named: "pixel"), title: "pixel"),
+            WebxdcApp(accountId: 0, chatId: 0, messageId: 4, image: UIImage(named: "checklist"), title: "checklist"),
+            WebxdcApp(accountId: 0, chatId: 0, messageId: 5, image: UIImage(named: "hello"), title: "hello"),
+            WebxdcApp(accountId: 0, chatId: 0, messageId: 7, image: UIImage(named: "webxdc"), title: "webxdc"),
+        ]
+
+        return UsedWebxdcEntry(date: Date(), apps: Array(apps.prefix(limit)))
+    }
+
+    func getSnapshot(in context: Context, completion: @escaping (UsedWebxdcEntry) -> Void) {
+        let entry = placeholder(in: context)
+        completion(entry)
+    }
+
+    func getTimeline(in context: Context, completion: @escaping (Timeline<Entry>) -> Void) {
+        /// ----------------------------------
+        /// **DO NOT** START dcAccounts.startIo() IN WIDGET
+        /// ----------------------------------
```

Comment:
i'd be a bit less broad and also try to answer "why":

```suggestion
        // no startIo() as this works only for one process, and widgets may run concurrently to the main app
```